static/specification.html

```diff
@@ -144,595 +204,516 @@ <h2>
-            report a warning via the console.
-            </li>
-            <li>If the Web Monetization <code>&lt;meta
-            name="monetization"&gt;</code> tag is well-formed, the browser
-            should extract the Payment Setup Endpoint (<a href=
-            "https://paymentpointers.org">Payment Pointer</a> or <a href=
-            "https://interledger.org/rfcs/0009-simple-payment-setup-protocol">
-              SPSP</a> Url).
-            </li>
-            <li>The <a>user agent</a> generates a fresh <strong>random</strong>
-            UUID (version 4) and uses this as a Monetization ID. This MUST be
-            regenerated upon dynamic changes to the meta tags and MUST be
-            unique per page load in order to avoid user tracking.
-            </li>
-            <li>The <a>user agent</a> passes these payment details to the
-            user's provider (see <a href="#payment-handler-flow">Payment
-            Handler Flow</a>).
-            </li>
-            <li>The provider resolves the Payment Setup Endpoint and begins to
-            pay. The provider MAY be acting from a different machine from the
-            user. Cookies and session information MUST NOT be carried with any
-            requests made to resolve the Payment Setup Endpoint, with the
-            exception of the Monetization ID.
-              <ul>
-                <li>With the <code>destination_account</code> and
-                <code>shared_secret</code> fetched from the SPSP query, a
-                [[[STREAM]]] connection is established. A single [[[STREAM]]]
-                is opened on this connection, and a positive SendMax is set.
-                </li>
-                <li>The provider SHOULD set their SendMax high enough that it
-                is never reached, and make payment adjustments by limiting
-                throughput.
-                </li>
-              </ul>
-            </li>
-            <li>Once the [[[STREAM]]] connection has fulfilled an ILP packet
-            with a non-zero amount, the provider notifies the browser:
-              <ul>
-                <li>The <a>user agent</a> sets
-                <code>document.monetization.state</code> to
-                <code>started</code>.
-                </li>
-                <li>This MUST occur before the `start` event is fired.
-                </li>
-                <li>This event is a {{ MonetizationStartEvent }} containing the
-                Payment Setup Endpoint and the Monetization ID as properties.
-                </li>
-                <li>The <a>user agent</a> also emits a {{ progress }} event
-                from <code>document.monetization</code>, corresponding to this
-                first packet.
-                </li>
-              </ul>
+        <li>Let |eventInitDict:MonetizationEventInit| be a
+        {{MonetizationEventInit}} dictionary, whose members are initialized to
+        match |packet|'s details.
```

Comment:
Here, I think we need to "canonicalize" the assetCode to uppercase and recommend ISO currency codes... basically, we need to copy/paste:
https://www.w3.org/TR/payment-request/#dom-paymentcurrencyamount-currency